### PR TITLE
Expanded docs of verify_results test helper function

### PR DESCRIPTION
**Pull request by matejak**

### CODE REVIEW

File: docs/contribute/testing.adoc [hunk 1]
```diff
@@ -215,10 +215,18 @@ a `RESULTS_FILE`:
...
 ----
 verify_results TYPE CONTENT_FILE RESULTS_FILE COUNT
+
+verify_results "def" test_probe_foo.xml results.xml 13
 ----
 
+The function extracts the actual test/definition result, and compares it with the respective test/definition comment.
+So for example, if the test contains `comment="true"`, the test passes only if result of the respective test is `true`,
+if `comment="false"`, a `false` result is expected.
+If the comment is missing or it has other value, it is assumed that the result shouldn't be neither `true` or `false`.
```
Comment (jan-cerny) [inline docs/contribute/testing.adoc]:
```suggestion
If the comment is missing or it has other value, it is assumed that the result should be neither `true` nor `false`.
```

File: docs/contribute/testing.adoc [hunk 2]
```diff
@@ -215,10 +215,18 @@ a `RESULTS_FILE`:
...
+
 NOTE: This function expects that the OVAL `TYPE` is numbered from `1` to `COUNT`
 in the `RESULTS_FILE`.
+Type is typically `def` or `tst` for definitions and tests respectively.
```
Comment (jan-cerny) [inline docs/contribute/testing.adoc]:
```suggestion
`TYPE` is typically `def` or `tst` for definitions and tests respectively.
```

File: docs/contribute/testing.adoc [hunk 3]
```diff
@@ -215,10 +215,18 @@ a `RESULTS_FILE`:
 [source,bash]
 ----
 verify_results TYPE CONTENT_FILE RESULTS_FILE COUNT
+
+verify_results "def" test_probe_foo.xml results.xml 13
 ----
 
+The function extracts the actual test/definition result, and compares it with the respective test/definition comment.
+So for example, if the test contains `comment="true"`, the test passes only if result of the respective test is `true`,
```
Comment (jan-cerny) [inline docs/contribute/testing.adoc]:
```suggestion
For example, if the test contains `comment="true"`, the test passes only if result of the respective test is `true`,
```

**Comment by matejak** [comment]
Thank you @jan-cerny for suggesting those fixes!

**Comment by jan-cerny** [comment]
@matejak Great! Thanks!

**Pull request merged**